pkg/virt-launcher/monitor_test.go

```diff
@@ -106,18 +105,24 @@ var _ = Describe("VirtLauncher", func() {
 	}
 
 	BeforeEach(func() {
+		cmdlineMatchStr = uuid.New().String()
+		processStarted = false
 		if !strings.Contains(fakeQEMUBinary, "../../") {
 			fakeQEMUBinary = filepath.Join("../../", fakeQEMUBinary)
 		}
 		gracefulShutdownChannel = make(chan struct{})
 		shutdownCallback := func(pid int) {
-			syscall.Kill(pid, syscall.SIGTERM)
+			// Don't send SIGTERM to the current process group (i.e. to PID 0). That will interrupt
+			// the test run.
+			Expect(pid).ToNot(BeZero())
```

Comment:
So there were zombie processes in the test env?

Reply:
There is a dedicated test for zombie:

https://github.com/kubevirt/kubevirt/blob/7e034450c10ad2a879db960b9912be2dff7ed9ce/pkg/virt-launcher/monitor_test.go#L145

Reply:
Got it thanks!